force-app/main/default/classes/GE_GiftEntryController.cls

```diff
@@ -186,6 +186,15 @@ public with sharing class GE_GiftEntryController {
         }
     }
 
+    @AuraEnabled(cacheable=true)
+    public static Boolean isRecurringGiftsEnabled() {
+        try {
+            return new GE_SettingsService().isRecurringGiftsEnabled();
```

Comment:
Can an instance of the service be stored as a local variable here so it doesn't get new'd up each time this method is called?